scripts/preprocessing/03-make_epochs.py

```diff
@@ -16,14 +16,25 @@
 import mne
 from mne.parallel import parallel_func
 from mne_bids import BIDSPath
-
+from autoreject import get_rejection_threshold
 import config
 from config import gen_log_message, on_error, failsafe_run
 
 logger = logging.getLogger('mne-bids-pipeline')
 
 
 @failsafe_run(on_error=on_error)
+
+def _get_global_reject_epochs(raw, events, event_id, tmin, tmax):
+    epochs = mne.Epochs(
+        raw, events, event_id=event_id, tmin=tmin, tmax=tmax,
+        proj=False, baseline=None, reject=None)
+    epochs.load_data()
+    epochs.pick_types(meg=True)
```

Comment:
```suggestion
```
We already pick the correct subset of channels in the very first preprocessing script, so picking again is not necessary.

Removing this line would then automatically make things work for EEG data too!